### ISSUE TITLE
Encrypted public data

### ISSUE THREAD

**Issue by brolund** [agical/friendsunhosted#15]
For the case someone wants all or part of their updates encrypted, updates are encrypted with a symmetric key, and all friends get a copy of that key, signed for their eyes only using the public key of the friend, and possibly the private key of the user for authenticity.

**Comment by johlrogge** [comment]
Yes, this touches on followers, confiremed, mutually confirmed and friends which I realize I should post an RFC on for discussion here

**Comment by jancborchardt** [comment]
Yeah, so then there would be a publishing option »public« or »friends only«.
